xml/snmp_sysdescr.xml

```diff
@@ -1796,6 +1796,20 @@ Copyright (c) 1995-2005 by Cisco Systems
     <param pos="1" name="os.version"/>
   </fingerprint>
 
+  <!--======================================================================
+                                Cumulus
+   =======================================================================-->
+
+  <fingerprint pattern="^Cumulus Linux (\S+) \(Linux Kernel ([^\)]+)\)">
+    <description>Cumulus Linux</description>
+    <example os.version="3.7.2" os.version.version="4.1.33-1+cl3u17">Cumulus Linux 3.7.2 (Linux Kernel 4.1.33-1+cl3u17)</example>
+    <param pos="0" name="os.vendor" value="Cumulus"/>
+    <param pos="0" name="os.device" value="Switch"/>
+    <param pos="0" name="os.product" value="Linux"/>
```

Comment:
Changing `os.product` to `Cumulus Linux` allows a CPE to be generated. Can we change the value here or should we add a remap?